config/bidderinfo.go

```diff
@@ -161,7 +161,8 @@ type SyncerEndpoint struct {
 	// startup:
 	//
 	//  {{.ExternalURL}} - This will be replaced with the host server's externally reachable http path.
-	//  {{.SyncerKey}}   - This will be replaced with the syncer key.
+	//  {{.SyncerKey}}   - This will be replaced with the bidder name.
```

Comment:
Nitpick: Let's remove the `{{.SyncerKey}}` macro from this comment as we're deprecating it in favor of the now more-accurate `{{.BidderName}}` macro.

Reply:
Done!